contents/handbook/engineering/incidents.md

```diff
@@ -30,6 +30,20 @@ Things that _shouldn’t_ be an incident
 
 > Planning some maintenance? Check the [announcements](/handbook/growth/comms/product-announcements) section instead.
 
+### Security-specific guidance
+
+Security incidents can have far-reaching consequences and should always be treated with urgency. 
+Some examples of security-related issues that warrant raising an incident include:
+
+- Unauthorized access to systems, data, or user accounts
+- Detection of malware, ransomware, or other malicious software on company systems
+- Suspicious activity on production infrastructure, such as unexpected user logins, privilege escalations, or resource consumption spikes
+- Discovery of exposed credentials, sensitive data, or secrets in logs, repositories, or public forums
+- Receiving a credible report of a vulnerability or exploit affecting company systems
+- Security-related issues that - if not handled - could have a negative impact on Posthog's reputation
```

Comment:
Personally I'd remove this line. It feels a little "we would look bad which is why we care"